### PR TITLE
Function_call in C++

### LINKED ISSUE #8285: Function call in c++
### Description

With my presentation, I would explain the function calls in C.

### Domain

C/CPP

### Type of Contribution

Audio, Video, Documentation

### Code of Conduct

- [X] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.

### PR DESCRIPTION
**Pull request by Anu2001-prog**
FUNCTION CALL IN C++ & ITS TYPES

## Description 📜

I have included the codes, documentation, and video of the presentation with audio explaining the function call and its types..
Fixes #8285 

## Type of change 📝

- [ ] Au#dio (Should be in mp3 format Includes speech clarity, Concise ,Low distortion)
- [x] Video (Animations, screen-recordings, presentations and regular explanatory films are all possibilties etc)
- [x] Documentation (Content Creation in the form of codes or tutorials)
- [ ] Other (If you choose other, Please mention changes below) 

<hr>

## Domain of Contribution 📊

- [ ] Android Dev #(Flutter)
- [ ] Android Dev #(Java)
- [ ] Android Dev #(Kotlin)
- [ ] Backend Dev #(Java)
- [ ] Backend Dev #(.NET)
- [ ] Backend Dev #(PHP)
- [ ] Backend Dev #(Python)
- [x] C/CPP
- [ ] Competitive #Programming
- [ ] Cyber #Security
- [ ] DS#A
- [ ] Data#base
- [ ] Datascience with #Python
- [ ] Datascience with #R
- [ ] Frontend Dev #HTML/CSS/JS
- [ ] Frontend Dev #React/Angular/Vue
- [ ] Go#lang
- [ ] Interview #Prep
- [ ] Java_#Domain
- [ ] JavaScript_#Domain
- [ ] ME#RN
- [ ] Machine #Learning
- [ ] Open #Source
- [ ] Python_#Domain
- [ ] Ru#st
- [ ] Statis#tics
- [ ] UI/#UX

<hr>

## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

<hr>

## Screenshots / Gif (Optional) 📸

<hr>

### CODE REVIEW

File: C_CPP/Fundamentals/Functions & Recursion and its types/Function_call.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
+# **Function call in c++**
+
+## What is a function call in c++?
+### What is a function call in c++?
```
Comment (Suvraneel) [inline C_CPP/Fundamentals/Functions & Recursion and its types/Function_call.md:4]:
```suggestion
## What is a function call in c++?
```
Redundant

File: C_CPP/Fundamentals/Functions & Recursion and its types/Function_call.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+ n=n+10;
+cout<<"The value of num in the called function ="<<n<<endl;
+}
+```
+*Output :*
+![call by reference](https://user-images.githubusercontent.com/84911349/137098429-a4b91a0a-c4ba-42d7-8ea6-57e5bc354161.png)
+ 
+ _We now, can clearly understand the implementation of function call in c++._
+ 
+ ### video explanation :
```
Comment (Suvraneel) [inline C_CPP/Fundamentals/Functions & Recursion and its types/Function_call.md:98]:
```suggestion
 ### Video Explanation :
```

**Review by Suvraneel — CHANGES_REQUESTED** [review]
@Anu2001-prog  Good work 

Changes :
1. Shift video to top of md file... Dikhna chahiye ekdumm upar
2. Add appropriate line to `C_CPP/Fundamentals/Functions & Recursion and its types/readme.md`

**Comment by Suvraneel** [comment]
@Anu2001-prog  LGTM

**Comment by Anu2001-prog** [comment]
thank you. @Suvraneel

**Pull request merged**